build/azure-pipeline.pre-release.yml

```diff
@@ -23,10 +23,6 @@ extends:
   template: azure-pipelines/extension/pre-release.yml@templates
   parameters:
     buildSteps:
-      - task: NodeTool@0
```

Comment:
Not sure why? I assume the latest node works too but why not the older one?

Reply:
Got the azure pipelines (upstream) to use the same version as VS Code, this way we don't have to specify this and everyone uses the right version